networkx/utils/misc.py

```diff
@@ -134,7 +134,9 @@ def make_str(x):
 
 def generate_unique_node():
     """ Generate a unique node label."""
```

Comment:
You could also add a `.. deprecated:: 2.6` to the docstring, which renders a nice warning in the html docs (if they're even generated for this function).

Reply:
I added this to all deprecated functions in misc.py
Some are in the reference section of the docs and some aren't.
Clearly the docs for utils are not up to date.